ginga/gtk3w/Widgets.py

```diff
@@ -2375,7 +2375,7 @@ def __init__(self, logger=None, settings=None):
         GObject.threads_init()
         # self._time_save = time.time()
 
-        for name in ('shutdown', ):
+        for name in ['close', 'shutdown']:
```

Comment:
No reason to not use tuple.
```suggestion
        for name in ('close', 'shutdown'):
```

Reply:
I read recently that tuples and lists are both stored as arrays (with associated Pyobject) in Python. Why do you prefer a tuple in this case? Just curious.

Reply:
Tuple is immutable, so I feel safer. 😆